User Guide.md

```diff
@@ -0,0 +1,130 @@
+This guide is written for users who want to buy and sell cryptocurrency via `xud`. It explains how to use [`xud-docker`](https://github.com/ExchangeUnion/xud-docker), the recommended and easiest way to get up and running.
+
+# Networks
+
+## Simnet
+This is where you want to start. It's quick and gives you a "look and feel". Private chains maintained by Exchange Union cloud instances, we automatically open channels to you and allocate you some precious simnet tokens. Trade against our bots. Consider restarting the setup once after starting for the first time (`down`,  `exit`, `bash ~/xud.sh`, `1`).
+
+Status: `live` | Setup time: `~15 mins` | Disk space: `5 GB`
+
+## Testnet
+bitcoin testnet 3, litecoin testnet 4, ethereum ropsten. Faucets: [t-BTC](https://coinfaucet.eu/en/btc-testnet/), [t-LTC](https://faucet.xblau.com/), [t-ETH 1](https://faucet.ropsten.be/) & [2](https://faucet.metamask.io/). Quite a bit of manual work to be done here.
+
+Status: `live` | Setup time: `~5-24h` | Disk space: `120 GB`
+
+## Mainnet
+Real money - only with #reckless hat.
+
+Status: `live` (with very low limits) | Setup time: `~1-3 days` | Disk space: `500 GB`
+
+## Regtest
+Producing blocks locally, mainly for development
+
+Status: `not available`| Setup time: `fast` | Disk space: `1 GB`
+
+# Requirements
+
+1. Linux, macOS or [Windows with WSL 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install).
+
+2. 8GB RAM (we saw some weird container states with less on testnet and mainnet). The more, the better. `geth` likes RAM. A lot.
+
+3. Run on a SSD. Based on painful experience: `geth` **cannot** catch up with the chain when running on a regular HDD. Read about it [here](https://medium.com/blockchain-studio/ethereum-client-geth-v1-9-0-released-whats-new-2b3de043ee16).
+
+4. `docker` >= 18.09 & `docker-compose` >= 1.24. Check with `docker --version` & `docker-compose --version`. If you do not have these installed yet, follow the official install instructions for [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/).
+
+5. Current user can run docker (without adding `sudo`). Test with `docker run hello-world`. If this fails, [follow these instructions](https://docs.docker.com/install/linux/linux-postinstall/).
+
+
+# How to run
+
+Start the environment with
+```bash
+curl https://raw.githubusercontent.com/ExchangeUnion/xud-docker/master/xud.sh -o ~/xud.sh
+bash ~/xud.sh
+ 
```

Comment:
```suggestion
```

Reply:
Deleted the space, but kept the line break to run `bash ~/xud.sh` when copy pasting the whole thing.